packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx

```diff
@@ -68,14 +63,20 @@ export class EditingItem extends BaseComponent<IEditingSelectedPeopleItemProps,
   }
 
   private _renderEditingSuggestions = (): JSX.Element => {
-    const onRenderFloatingPicker = this._onRenderFloatingPicker;
-    return onRenderFloatingPicker({
-      componentRef: this._editingFloatingPicker,
-      onChange: this._onSuggestionSelected,
-      inputElement: this._editingInput,
-      selectedItems: [],
-      ...this._floatingPickerProps
-    });
+    const FloatingPicker = this.props.onRenderFloatingPicker;
+    const floatingPickerProps = this.props.floatingPickerProps;
+    if (!FloatingPicker || !floatingPickerProps) {
+      return <></>;
+    }
+    return (
+      <FloatingPicker
```

Comment:
Would you still want to call this.props.onRenderFloatingPicker even when `floatingPickerProps` is undefined?